src/css/style.css

```diff
@@ -67,22 +67,28 @@
     padding: 6px 11px;
 }
 
-.dp-consent-button--primary {
-    color: #fff;
-    background-color: #006fd0;
+.dp-consent-button--primary-outline {
```

Comment:
Although the PR changes the button appearance to an outline button, i think this must not be reflected by the class name. It is still a primary button, and as there are no other button variants used (afaic), i'd keep the naming as it was before.

Reply:
Done in 0270b4418af0e36db79c92d6e39e2ea58460ad0f